cmd/query/app/token_propagation_handler.go

```diff
@@ -26,21 +27,30 @@ import (
 func bearerTokenPropagationHandler(logger *zap.Logger, h http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
+		logger.Info("Propagating bearer token")
+		log.Print(r)
```

Comment:
We don't use log package

Reply:
Ups it was not my intend to have this here.